etl/steps/data/garden/countries/2023-09-25/gleditsch.countries.json

```diff
@@ -0,0 +1,228 @@
+  "Nepal": "Nepal",
+  "Netherlands": "Netherlands",
+  "New Zealand": "New Zealand",
+  "Nicaragua": "Nicaragua",
+  "Niger": "Niger",
+  "Nigeria": "Nigeria",
+  "Norway": "Norway",
+  "Oman": "Oman",
+  "Pakistan": "Pakistan",
+  "Palau": "Palau",
+  "Panama": "Panama",
+  "Papua New Guinea": "Papua New Guinea",
+  "Paraguay": "Paraguay",
+  "Peru": "Peru",
+  "Philippines": "Philippines",
+  "Poland": "Poland",
+  "Portugal": "Portugal",
+  "Qatar": "Qatar",
+  "Rwanda": "Rwanda",
+  "Saint Kitts and Nevis": "Saint Kitts and Nevis",
+  "Saint Lucia": "Saint Lucia",
+  "Saint Vincent and the Grenadines": "Saint Vincent and the Grenadines",
+  "San Marino": "San Marino",
+  "Saudi Arabia": "Saudi Arabia",
+  "Senegal": "Senegal",
+  "Serbia": "Serbia",
+  "Seychelles": "Seychelles",
+  "Sierra Leone": "Sierra Leone",
+  "Singapore": "Singapore",
+  "Slovakia": "Slovakia",
+  "Slovenia": "Slovenia",
+  "Solomon Islands": "Solomon Islands",
+  "Somalia": "Somalia",
+  "South Africa": "South Africa",
+  "South Ossetia": "South Ossetia",
+  "South Sudan": "South Sudan",
+  "Spain": "Spain",
+  "Sudan": "Sudan",
+  "Swaziland": "Eswatini",
+  "Sweden": "Sweden",
+  "Switzerland": "Switzerland",
+  "Syria": "Syria",
+  "Taiwan": "Taiwan",
+  "Tajikistan": "Tajikistan",
+  "Thailand": "Thailand",
+  "Togo": "Togo",
+  "Tonga": "Tonga",
+  "Trinidad and Tobago": "Trinidad and Tobago",
+  "Tunisia": "Tunisia",
+  "Turkmenistan": "Turkmenistan",
+  "Tuvalu": "Tuvalu",
+  "Uganda": "Uganda",
+  "Ukraine": "Ukraine",
+  "United Arab Emirates": "United Arab Emirates",
+  "United Kingdom": "United Kingdom",
+  "United States of America": "United States",
+  "Uruguay": "Uruguay",
+  "Uzbekistan": "Uzbekistan",
+  "Vanuatu": "Vanuatu",
+  "Venezuela": "Venezuela",
+  "Yugoslavia": "Yugoslavia",
+  "Zambia": "Zambia",
+  "Zanzibar": "Zanzibar",
+  "Antigua & Barbuda": "Antigua and Barbuda",
+  "Belarus (Byelorussia)": "Belarus",
+  "Bosnia-Herzegovina": "Bosnia and Herzegovina",
+  "Burkina Faso (Upper Volta)": "Burkina Faso",
+  "Cambodia (Kampuchea)": "Cambodia",
+  "Congo, Democratic Republic of (Zaire)": "Democratic Republic of Congo",
+  "Cote D\u0092Ivoire": "Cote d'Ivoire",
+  "Federated States of Micronesia": "Micronesia (country)",
+  "Iran (Persia)": "Iran",
+  "Korea": "Korea (former)",
+  "Macedonia (Former Yugoslav Republic of)": "North Macedonia",
+  "Myanmar (Burma)": "Myanmar",
+  "Rumania": "Romania",
+  "Samoa/Western Samoa": "Samoa",
+  "Sri Lanka (Ceylon)": "Sri Lanka",
+  "Surinam": "Suriname",
+  "S\u00e3o Tom\u00e9 and Principe": "Sao Tome and Principe",
+  "Tanzania/Tanganyika": "Tanzania",
+  "Vietnam, Democratic Republic of": "Vietnam",
+  "Yemen (Arab Republic of Yemen)": "Yemen",
+  "Zimbabwe (Rhodesia)": "Zimbabwe",
+  "Korea, People's Republic of": "North Korea",
+  "Baden": "Baden",
+  "Bavaria": "Bavaria",
+  "German Democratic Republic": "German Democratic Republic",
+  "German Federal Republic": "German Federal Republic",
```

Comment:
Aren't the OWID names for these countries `East Germany` and `West Germany`, respectively?

Reply:
Totally, nice catch!